blazor/datagrid/filtering.md

```diff
@@ -124,192 +438,1138 @@ LessThanOrEqual |Checks whether the value is less than or equal to the specified
+            this.CustomerID = CustomerID;
+            this.Freight = Freight;
+            this.ShipCity = ShipCity;
+            this.ShipCountry = ShipCountry;
+        }
+
+        public static List<OrderData> GetAllRecords()
+        {
+            if (Orders.Count() == 0)
+            {
+                int OrderID = 10247;
+                for (int i = 1; i < 7; i++)
+                {
+                    Orders.Add(new OrderData(OrderID + 1, "VINET", 32.38, "Reims", "France"));
+                    Orders.Add(new OrderData(OrderID + 2, "TOMSP", 11.61, "Münster", "Germany"));
+                    Orders.Add(new OrderData(OrderID + 3, "HANAR", 65.83, "Rio de Janeiro", "Brazil"));
+                    Orders.Add(new OrderData(OrderID + 4, "VICTE", 45.78, "Lyon", "Belgium"));
+                    Orders.Add(new OrderData(OrderID + 5, "SUPRD", 98.6, "Charleroi", "Switzerland"));
+                    Orders.Add(new OrderData(OrderID + 6, "HANAR", 103.45, "Lyon", "Venezuela"));
+                    Orders.Add(new OrderData(OrderID + 7, "CHOPS", 103.45, "Rio de Janeiro", "Austria"));
+                    Orders.Add(new OrderData(OrderID + 8, "RICSU", 112.48, "Münster", "Mexico"));
+                    Orders.Add(new OrderData(OrderID + 9, "WELLI", 33.45, "Reims", "USA"));
+                    OrderID += 9;
+                }
+            }
+            return Orders;
+        }
+
+        public int? OrderID { get; set; }
+        public string CustomerID { get; set; }
+        public double? Freight { get; set; }
+        public string ShipCountry { get; set; }
+        public string ShipCity { get; set; }
+    }
+{% endhighlight %}
+{% endtabs %}
+
+{% previewsample "https://blazorplayground.syncfusion.com/embed/hjhpZCiFKFFRoZzt?appbar=false&editor=false&result=true&errorlist=false&theme=bootstrap5" %}
+
+**2.Using the properties in the  FilteredEventArgs object**
+
+Alternatively, you can use the properties available in the  FilteredEventArgs object to obtain the filter record details.
+
+* [ColumnName](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grids.FilteredEventArgs.html#Syncfusion_Blazor_Grids_FilteredEventArgs_ColumnName): This property returns the column name that is currently filtered.
+
+* [FilterPredicates](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grids.FilteredEventArgs.html#Syncfusion_Blazor_Grids_FilteredEventArgs_FilterPredicates): This property returns the object containing filter predicate model details that is currently filtered.
+
+To access these properties, you can use the [Filtered](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grids.GridEvents-1.html#Syncfusion_Blazor_Grids_GridEvents_1_Filtered) event handler as shown below:
+
+```cshtml
+  public async Task FilteredHandler(FilteredEventArgs args)
+    {
+        var columnname = args.ColumnName;
+        var FilterPredicates = args.FilterPredicates;
+    }
+```
+
+### Get filtered records when using remote data source
```

Comment:
mmove this topics as a second topic below 1st topic. 

move below to 1.Using the getFilteredRecords() method and change that to 1.Using the getFilteredRecords() method for remote data